migrateDbAndStartServer.sh

```diff
@@ -1,30 +1,60 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Wait until the relational database-server up and running
 waited=0
-until node ./utils/testSequelizeDbServerAvailable.js
+until node ./utils/testDatabaseConnectionsAvailable.js 1>/dev/null
 do
-	if [ $waited == 240 ]; then
-		echo -e '\nERROR: Time out reached while waiting for relational database server to be available.\n'
-		exit 1
-	fi
-	sleep 2
-	waited=$(expr $waited + 2)
+  if [ $waited == 240 ]; then
+    echo -e '\nERROR: Time out reached while waiting for relational database server to be available.\n'
+    exit 1
+  fi
+  sleep 2
+  waited=$(expr $waited + 2)
+done
+
+
+# Read config and migrate/seed databases
+CONFIG="./config/data_models_storage_config.json"
+SEQUELIZE="./node_modules/.bin/sequelize"
+
+jq -r 'keys[] as $k | "\($k):\(.[$k] | .storageType)"' < $CONFIG |
```

Comment:
Like we agreed: add scripts folder and the appropriate javascript script, that does this